ISM43362Interface.h

```diff
@@ -135,29 +121,6 @@ class ISM43362Interface : public NetworkStack, public WiFiInterface
      */
     virtual int scan(WiFiAccessPoint *res, unsigned count);
 
-    /** Translates a hostname to an IP address with specific version
-     *
-     *  The hostname may be either a domain name or an IP address. If the
-     *  hostname is an IP address, no network transactions will be performed.
-     *
-     *  If no stack-specific DNS resolution is provided, the hostname
-     *  will be resolve using a UDP socket on the stack.
-     *
-     *  @param address  Destination for the host SocketAddress
-     *  @param host     Hostname to resolve
-     *  @param version  IP version of address to resolve, NSAPI_UNSPEC indicates
-     *                  version is chosen by the stack (defaults to NSAPI_UNSPEC)
-     *  @return         0 on success, negative error code on failure
-     */
-    using NetworkInterface::gethostbyname;
```

Comment:
Both the NetworkStack and NetworkInterface provide their implementations of gethostbyname...
If you remove the explicit declaration of which one you want to use, will this code still compile? And if so, which implementation is going to be used?
One way out would be to prepend each call to gethostbyname with an explicit BaseClass:: specifier... Is this the plan?